geode-core/src/main/java/org/apache/geode/internal/admin/SSLConfig.java

```diff
@@ -33,11 +34,11 @@
   private String ciphers = DistributionConfig.DEFAULT_SSL_CIPHERS;
   private boolean requireAuth = DistributionConfig.DEFAULT_SSL_REQUIRE_AUTHENTICATION;
   private String keystore = DistributionConfig.DEFAULT_SSL_KEYSTORE;
-  private String keystoreType = DistributionConfig.DEFAULT_CLUSTER_SSL_KEYSTORE_TYPE;
+  private String keystoreType = KeyStore.getDefaultType();
```

Comment:
I notice that `DistributionConfig.DEFAULT_CLUSTER_SSL_KEYSTORE_TYPE` is marked as deprecated. Is this its replacement?